homeassistant/components/sighthound/image_processing.py

```diff
@@ -69,6 +79,8 @@ def __init__(self, api, camera_entity, name):
         self._state = None
         self._image_width = None
         self._image_height = None
+        if save_file_folder:
```

Comment:
We should always set all instance attributes in init method. At minimum default to `None`. This is general Python best practice.

Reply:
indeed!